README.md

```diff
@@ -130,6 +130,14 @@ running the connector.
 
 Optional string parameter to specify the port of the Cumulocity server. Default: "8883"
 
+#### Certificate URL (CertificateUrl)
+
+Optional string parameter to specify the URL of the Cumulocity server's certificate. 
+Default: "https://certs.godaddy.com/repository/gdroot-g2.crt"
```

Comment:
Is this something hosted by Cumulocity or are we hosting something on godaddy?

Reply:
This is the certificate used by Cumulocity cloud, and it is publicly downloadable from GoDaddy itself.